src/webui/routes/register.jl

```diff
@@ -38,7 +38,7 @@ function register(r::HTTP.Request)
 
     # Get the (Julia)Project.toml, and make sure it is valid.
     toml = gettoml(u.forge, repo, ref, subdir)
-    toml === nothing && return json(400; error="(Julia)Project.toml was not found")
+    toml === nothing && return json(400; error="(Julia)Project.toml was not found in repo `$repo` with ref `$ref` and subdir `$subdir`")
```

Comment:
`repo`, `ref`, and `subdir` are untrusted inputs provided by the user, so do we need to escape or sanitize them here?